tasks/generate-placeholder-file-for-extension-styles.js

```diff
@@ -0,0 +1,6 @@
+const fs = require('fs');
+
+fs.writeFileSync(
+    'styles/extension-styles.generated.css',
```

Comment:
wouldn't it be possible to handle that on webpack level? we can create entry point with multiple input files

Reply:
I'm not sure it's possible, since the locations of extensions and their CSS files are specified only in the root repo. It would also probably not work well with the goal to distribute client-core to npm pre-built(instead of source as we do now), which would require isolating its build process from root repo and planning/analytics modules.